### PR TITLE
Rename azure_hana_media.yaml to hana_media.yaml

### PR DESCRIPTION
**Pull request by mpagot**
Bump the config.yaml apiver to 2 and enforce apiver check
Improve error msg about folder structure

### CODE REVIEW

**Review by lpalovsky — APPROVED** [review]
LGTM
- is there a VR to check if it breaks something?

File: scripts/qesap/test/unit/test_qesap_configure.py [hunk 1]
```diff
@@ -278,7 +278,7 @@ def test_configure_tfvars_overwrite_variables(configure_helper):
 
 def test_configure_create_ansible_vars(configure_helper, config_yaml_sample):
     """
-    Test that 'configure' write an azure_hana_media.yaml file in
+    Test that 'configure' write an hana_media.yaml file in
```
Comment (alvarocarvajald) [inline scripts/qesap/test/unit/test_qesap_configure.py:281]: s/an/a

File: scripts/qesap/test/unit/test_qesap_configure.py [hunk 2]
```diff
@@ -292,12 +292,12 @@ def test_configure_create_ansible_vars(configure_helper, config_yaml_sample):
 
 def test_configure_ansible_vars_content(configure_helper, config_yaml_sample):
     """
-    Test that 'configure' write an azure_hana_media.yaml with
+    Test that 'configure' write an hana_media.yaml with
```
Comment (alvarocarvajald) [inline scripts/qesap/test/unit/test_qesap_configure.py:295]: s/an/a

**Review by alvarocarvajald — APPROVED** [review]
LGTM. Just a couple of minor typos.

File: scripts/qesap/test/unit/test_qesap_configure.py (hunk 1 above)
Comment (mpagot) [reply scripts/qesap/test/unit/test_qesap_configure.py:281]: Thanks, I'll fix them in a different commit

**Pull request merged**